### PR TITLE
Feature modal component

### PR DESCRIPTION
**Pull request by ghost**
* TASK
   * Convert modal into a separate component

* Changes
   * Creation of ModalComponent file
   * Use of Ref forwarding technique to pass data from children to parent component (ModalComponent.js)
   * Adapt code to work with the new implementation. (Form.js)

### CODE REVIEW

File: src/Form.js [hunk 1]
```diff
@@ -97,7 +100,7 @@ export default class Form extends React.Component {
   modal = React.createRef();
 
   render() {
-    const { fullName, position } = this.state;
+    /* const { fullName, position } = this.state; */
```
Comment (infojasyrc) [inline src/Form.js]: Why is this commented?

**Review by infojasyrc — CHANGES_REQUESTED** [review]
Please review feedback and also update this branch with development to see only changes regarding this pull request.

File: src/Form.js (hunk 1 above)
Comment (ghost) [reply src/Form.js]: Since I moved the modal part to a separate component this assignment is no longer needed.

**Review by infojasyrc — APPROVED** [review]
👍

**Pull request merged**